src/core/os/amdgpu/amdgpuDevice.h

```diff
@@ -883,6 +883,8 @@ class Device : public Pal::Device
 
     bool HasFp16DisplaySupport();
 
+    bool HasRgba16DisplaySupport();
```

Comment:
If possible, I'd like HasFp16DisplaySupport() and HasRgba16DisplaySupport() to both be const functions.

Reply:
Adding const keywords a la?
bool HasRgba16DisplaySupport() const;

I can change that.

Reply:
great. there is another review result from internal, please take a look
Have you checked this in X/wayland compositing mode? I suppose some configurations are needed for these window systems to handle this format.

Reply:
All done in updated pull request.

Wrt. X/Wayland: We are good there. This pal code for enumeration of supported display formats is not used at all for windowing systems other than direct display fullscreen mode, ie. it is only used for Vulkan/WSI/Display. Cfe. the XGL code which calls these routines. In fact, no format enumeration is done at all for X11 or Wayland. Instead code in XGL only exposes a hard-coded list of the two most basic formats RGBA8 unorm and RGBA8 srgb. The code-path for needsWorkaround = true is used. See here:

https://github.com/GPUOpen-Drivers/xgl/blob/7e2070be478a90852ca88c2aee36406b9d19c861/icd/api/vk_physical_device.cpp#L3172